opr/assetList.go

```diff
@@ -71,9 +107,8 @@ func (o OraclePriceRecordAssetList) MarshalJSON() ([]byte, error) {
 		// add key
 		kEscaped := strings.Replace(k, `"`, `\"`, -1)
 		s = s + `"` + kEscaped + `":`
-		// add value
-		v := o[k]
-		vBytes, err := json.Marshal(v)
+		// add value as float.
+		vBytes, err := json.Marshal(o.Value(k))
```

Comment:
in v2 we're saving the value as uint64 on chain but here it's encoding the v2 json as a float.

Reply:
Meaning encode them as uint64s?

Reply:
yes

Reply:
@WhoSoup After some conversation, are you ok with keeping this a float value in the JSON?